adding a page with common CLI commands

josephalsup opened the pull request:
adding in cli-commands.html to course-content

barrycumbie commented:
@josephalsup this will be super useful. Need to:

1. use Google style guide (2 spaces vs. full indent0
1. Google style guide (no terminating ```</li>``` or not? go look and see
1. Did you validate this on Nu? That'd be nice to know for someone like ol @barrycumbie reviewing your code
1. how bout a better PR name (iss73 is meaningless to me)...the semantic approach: adding a page with common CLI cmds
1. look at all those awesome github Issue/PR features over there 👉🏼 Oh wait, you did! Linked to an issue 👏🏼 
  1. what about assigning yourself to this...get full credit of this PR as the "assignee"
1. For those cmds you have...need a bit more... its not just ```mkdir``` its ```mkdir foldername```

```shell
# to create a folder called repos in your current directory 
mkdir repos/ 
# note, the whack (/) isn't always needed but helps for clarity to see that this is a folder
# tips on naming folders? lowercase, snake-case, use plural too: imgs vs img sincs a buncha images are in there. 
```

josephalsup commented:
* I fixed my indents so that they are all 2 spaces now.
* Section 2.2.1 and 2.3.3 of the Google Style Guide show examples without the ending </li> tag and section 3.1.7 states that you can remove optional tags such as </li> but it is optional.
    * [Google Style Guide] (https://google.github.io/styleguide/htmlcssguide.html)
* I validated my page with Nu, do I need to add a link in the comments like in Dev01? Or in the page itself?
* I added more detail to my list of commands so that they are more usable.

barrycumbie commented:
> * I fixed my indents so that they are all 2 spaces now.
😎 
> * Section 2.2.1 and 2.3.3 of the Google Style Guide show examples without the ending
> tag and section 3.1.7 states that you can remove optional tags such as but it is optional.
> * https://google.github.io/styleguide/htmlcssguide.html
ah, optional. Now I know...I'll let you do as you please. 
> * I validated my page with Nu, do I need to add a link in the comments like in Dev01? Or in the page itself?
I trust you...you just have to tell me...Nu approved ✅ (no link required) 
> * I added more detail to my list of commands so that they are more usable.
super 😎
